src/fifelse.c

```diff
@@ -429,28 +429,28 @@ SEXP psumR(SEXP na, SEXP args) {
 
 SEXP pprodR(SEXP na, SEXP args) {
   if (!IS_TRUE_OR_FALSE(na)) {
-    error("Argument 'na.rm' must be TRUE or FALSE and length 1.");
+    error(_("Argument 'na.rm' must be TRUE or FALSE and length 1."));
   }
   const int n=length(args);
   if (n <= 1) {
-    error("Please supply at least 2 arguments. %d argument supplied.", n);
+    error(_("Please supply at least 2 arguments. %d argument supplied."), n);
   }
   SEXPTYPE anstype = TYPEOF(SEXPPTR_RO(args)[0]);
   const int64_t len0 = xlength(SEXPPTR_RO(args)[0]);
   if (anstype != INTSXP && anstype != REALSXP) {
-    error("Argument 1 is of type %s. Only integer and double types are supported.", type2char(anstype));
+    error(_("Argument 1 is of type %s. Only integer and double types are supported."), type2char(anstype));
```

Comment:
for translation, better to use %d here as well, then only one message to translate instead of 2

Reply:
Sorry Michael, I did not understand what you mean.

Reply:
each _unique_ item wrapped in _() will generate a message to be translated

one with Argument 1, another with Argument %d -- twice as much work for our translation team

vs using Argument %d for both, and hard-coding 1 to fill that in the former case; looks same to user, very similar in code, and easier for translators